docs/overview/why-saleor/extensibility.mdx

```diff
@@ -1,48 +1,81 @@
 ---
-title: Extensibility model
+title: Extensible
 ---
 
-### API Only
+Saleor's goal is to map complex commerce requirements to maintainable architectures. We recognize four key areas that define how easy or hard it is to design and maintain a commerce deployment: modeling, automation, delegation, and observability.
 
-- **Webhooks.**
-  You can subscribe to more than a hundred [events](/developer/extending/webhooks/overview.mdx)
-  to receive data from Saleor.
+But before diving into the details of those aspects, we must clarify one thing.
 
-- **API extensions.**
-  Enable behavior customization, for example, calculating shipping costs based on the package dimensions.
-  [Learn more](developer/extending/webhooks/synchronous-events/overview.mdx)
+## Tech Agnostic
 
-- **Dashboard extensions.** [Extend admin](/developer/extending/apps/overview.mdx) functionality using any front-end stack embeddable into _iframes_.
+While Saleor's engine is written in Python, it makes almost no assumptions about your tech stack. Saleor's API and the extension mechanism utilize GraphQL transported over either HTTP or message queues. Hence, the only requirement we impose is that your code be able to make or receive HTTP requests.
 
-- **Granular permission control.** [Granular permissions](/developer/permissions.mdx) for access tokens.
+Whether you work with Python, TypeScript, PHP, Java, C#, Elixir, or Rust, you can integrate with Saleor. And chances are there is already a GraphQL client library you can use with enough tooling to provide a fantastic developer experience.
 
-- **GraphQL oriented.** Provide GraphQL queries to shape the data provided via Webhooks.
+At Saleor, we always strive to build on existing standards. This way, you get the best interoperability and the benefit of existing tools and experience, and no one has to train specifically to become a "Saleor developer."
 
-### Why not plugin architecture?
+## Modeling
 
-The plugin architecture is popular among open-source projects; classic examples would be Magento or Shopify.
-Such a model has direct coupling with the platform,
-which can be convenient to get started quickly but can also lead to many issues.
+Extensibility starts with mapping your business needs to data shapes. While there are stores where the product is nothing but a name, a picture, and a two-sentence description, we recognize that it's rarely the case.
 
-**Benefits of decoupled architecture:**
+Saleor's product type system allows you to create named schemas for different product types, each having a different set of fields of various types. This guarantees that your books always have a page count that is a number and that your drinks have a list of ingredients, each linked to its own entity.
```

Comment:
```suggestion
Saleor's product type system allows you to create named schemas for different product types, each having a different set of attributes of various types. This guarantees that your books always have a page count that is a number and that your drinks are linked with their ingredients.
```

**style (non-blocking):**
The rationale behind the first proposed change is to use the vocabulary that may be already familiar to Saleor users.

With the second one, I am trying to propose a bit simpler structure. To me, it currently reads like there are three things: drinks, ingredients and some entity. But maybe we want to emphasize that ingredient is a separate entity.

Reply:
I agree with the "attributes" change, but the second change is semantically different. My version invokes a many-to-many relationship through an orderable list of references; yours implies a many-to-one relationship.